doc/conf_ref.rst

```diff
@@ -325,6 +325,15 @@ configuration file by your distribution to override the DNF defaults.
     disable automatic metadata synchronizing. The default corresponds to three
     hours. The value is rounded to the next commenced hour.
 
+.. _module_obsoletes-label:
+
+``module_obsoletes``
+    :ref:`boolean <boolean-label>`
+
+    This option controls whether dnf should apply modular obsoletes. If enabled dnf
+    checks on each transaction if there are any active modular obsoletes and if
+    possible it tries to apply them in the transaction.
```

Comment:
I'm wondering if it is necessery to describe how dnf does it and if there can be more than one transaction during one dnf run. Is the second sentence even needed? Because it adds these pieces of information:
- it may not be possible to apply modular obsoletes
- only "active" modular obsoletes are applied
- modular obsoletes is something that is applied in transactions (which I would assume anyway based on my experience with normal obsoletes)

How about just extend the first sentence: "This option controls whether dnf should apply active modular obsoletes when possible."

But maybe I understand it wrong, because I know nothing of modular obsoletes and most notably I have no idea what modular obsoletes are "active". Btw. is this documented somewhere?

Reply:
That is a very good point I don't think its documented, it is just a concept I have for obsoletes which either don't have eol_date or whose eol_date is set in the past. eol_date is a field in the [metadata format](https://github.com/fedora-modularity/libmodulemd/blob/main/yaml_specs/modulemd_obsoletes_v1.yaml) and if specified it refers to a date from which the module is considered eol. It probably doesn't make much sense here and it should be in libmodulemd docs or some "creating a modular obsolete" docs.

I have updated it to just: "This option controls whether dnf should apply modular obsoletes when possible."
Thanks!